bucket/hexchat.json

```
@@ -26,13 +22,24 @@
         ]
     ],
     "persist": "config",
+    "checkver": {
+        "github": "https://github.com/hexchat/hexchat"
+    },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://dl.hexchat.net/hexchat/HexChat%20$version%20x64.exe"
+                "url": "https://dl.hexchat.net/hexchat/HexChat%20$version%20x64.exe",
+                "hash": {
+                    "url": "https://hexchat.github.io/downloads.html",
+                    "regex": "$sha256\\s+HexChat $version x64"
```

There is no reason to combine hard coded whitespace with `\s`
```suggestion
                    "regex": "$sha256\\s+HexChat\\s+$version\\s+x64"
```